docs/pages/get_started/designers.js

```diff
@@ -44,17 +44,13 @@ In your first couple of weeks at Pinterest, you will receive an invite from your
           description={`
 **Please note: The following plugins are only available for Pinterest employees.**
 
-[**Gestalt docs for Figma**](https://www.figma.com/community/plugin/977755389228415646/Gestalt-docs-for-Figma-(Beta))
-View the Gestalt documentation without leaving Figma!
-
-[**Pinsert**](https://www.figma.com/community/plugin/763812093925718603/Pinsert)
-Insert Pins, Boards, and Profiles from Pinterest.
+[![promo image for the pinterest design figma plugin](https://i.pinimg.com/originals/2b/4b/b4/2b4bb4f0c0e82ddc8f37917c82738b7d.png)](https://www.figma.com/community/plugin/1215463263194174399)
```

Comment:
Also can we place the image below the text?

Reply:
<img width="651" alt="Screen Shot 2023-03-29 at 1 20 00 PM" src="https://user-images.githubusercontent.com/50343812/228657940-581ed563-175a-48a0-9aae-8fea4761843f.png">
Hmm... From my eye, that muddies the water as to which plugin this image refers to. 

The intention by this lockup was to follow the same structure we use for component images:
<img width="409" alt="Screen Shot 2023-03-29 at 1 21 24 PM" src="https://user-images.githubusercontent.com/50343812/228658320-b4d44570-feb8-45f9-ae98-2e1a89032413.png">

Reply:
MMMm

Reply:
I like below the text too! Because in the component image, we treat it as a visual array, and it's the main focus.

In this case, we're reading top to bottom, so I think images inline with the text, than starting the paragraph with the image works better